kitchen.yml

```diff
@@ -1,27 +1,27 @@
+---
 driver:
   name: vagrant
 
 provisioner:
   name: chef_zero
-  deprecations_as_errors: true
-  chef_license: accept-no-persist
+  product_name: chef
+  product_version: <%= ENV['CHEF_VERSION'] || 'latest' %>
+  install_strategy: once
+
+client_rb:
+  treat_deprecation_warnings_as_errors: true
 
 verifier:
   name: inspec
 
 platforms:
-  - name: amazonlinux
-    driver_config:
-      box: mvbcoding/awslinux
   - name: centos-6
   - name: centos-7
```

Comment:
we have a centos-8 box now too

Reply:
I can add it but trying to stick to versions defined here: https://sous-chefs.org/docs/best-practices/